certify-service/src/test/java/io/mosip/certify/services/KeymanagerLibSignerTest.java

```diff
@@ -60,32 +57,25 @@ public class KeymanagerLibSignerTest {
 
     @Before
     public void setup() {
+        ReflectionTestUtils.setField(signer, "hostedKey", "https://example.com/sample.pub.key.json/");
+        ReflectionTestUtils.setField(signer, "signProps", new RsaProofSignature2018());
     }
 
     @Test
     public void testPerformSuccess_VC2() {
         // Mock Templated VC and Key Manager Input
         String VCs[] = new String[]{VC_1, VC_2};
         for (String templatedVC : VCs) {
-
-            Map<String, String> keyMgrInput = new HashMap<>();
-            keyMgrInput.put(KeyManagerConstants.PUBLIC_KEY_URL, "https://example.com/sample.pub.key.json/");
-            keyMgrInput.put(KeyManagerConstants.KEY_APP_ID, KeyManagerConstants.CERTIFY_MOCK_RSA);
-            keyMgrInput.put(KeyManagerConstants.KEY_REF_ID, KeyManagerConstants.EMPTY_REF_ID);
-            keyMgrInput.put(KeyManagerConstants.VC_SIGN_ALGO, SignatureAlg.RSA_SIGNATURE_SUITE);
-            keyMgrInput.put(KeyManagerConstants.KEYMGR_SIGN_ALGO, JWSAlgorithm.RS256.getName());
-
             // Mock Signature Service Response
             JWTSignatureResponseDto jwsSignedData = new JWTSignatureResponseDto();
             jwsSignedData.setJwtSignedData("mocked-jws");
             when(signatureService.jwsSign(any())).thenReturn(jwsSignedData);
             // Perform the test
-            VCResult<JsonLDObject> vcResult = signer.perform(templatedVC, keyMgrInput);
+            VCResult<JsonLDObject> vcResult = signer.perform(templatedVC);
 
             // Assertions
-            Assert.assertNotNull(vcResult);
+            assert vcResult != null;
             JsonLDObject credential = vcResult.getCredential();
-            Assert.assertNotNull(credential);
             Assert.assertNotNull(credential.getJsonObject().get(VCDMConstants.PROOF));
             Assert.assertNotNull(vcResult.getCredential().getJsonObject().containsKey("proof"));
```

Comment:
I removed multiple assert statements as per a good suggested best practice to have one assert per test method.